twitter_retweet_block.py

```diff
@@ -10,30 +10,32 @@
 class TwitterRetweet(TwitterRestBase):
 
     version = VersionProperty("1.0.0")
-    id = Property(default='{{$id}}', title='Tweet ID')
+    tweet_id = Property(default='{{$tweet_id}}', title='Tweet ID')
 
     def process_signals(self, signals):
-        for s in signals:
+        for signal in signals:
             try:
-                id = self.id(s)
+                tweet_id = self.tweet_id(signal)
```

Comment:
these refs to `id` inside `process_signals` are actually safe, it's refs to `self.id` that are an issue. This change is fine but unnecessary. I guess it looks/reads a little better tho so if you like it better, I approve.